### PR TITLE
Yet more posts

### PR DESCRIPTION
**Pull request by pwalczysko**
See https://trello.com/c/aTdBNblP/18-news-migration, checklist ``Petr (second batch)``

### CODE REVIEW

File: _posts/2014-08-06-omero-bio-formats-5-0-3.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+Further details are available on https://trac.openmicroscopy.org.uk/ome/milestone/5.0.3
+
+To see details of the actual code changes and the specific issues they are addressing, refer to the GitHub milestone pages for OMERO and Bio-Formats.
+
+The software is available from
+[archived downloads](http://downloads.openmicroscopy.org/omero/5.0.3/)
+and
+[archived downloads](http://downloads.openmicroscopy.org/bio-formats/5.0.3/)
+
+For information on upgrading your server, see http://www.openmicroscopy.org/site/support/omero5/sysadmins/server-upgrade.html.
```
Comment (sbesson) [inline _posts/2014-08-06-omero-bio-formats-5-0-3.md]: URL should be formatted using Markdown (`[url](url)`)

File: _posts/2014-08-06-omero-bio-formats-5-0-3.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+Any problems or comments, please use the OME Forums or mailing lists:
+
+http://www.openmicroscopy.org/site/community
```
Comment (sbesson) [inline _posts/2014-08-06-omero-bio-formats-5-0-3.md]: `Any problems or comments, please use the [OME Forums or mailing lists]({{ site.baseurl }}/support/)`

File: _posts/2014-09-02-omero-bio-formats-5-0-4.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+
+Full details are available on the OMERO Github milestone[1] and the Bio-Formats Github milestone[2] pages.
+
+The software is available from
+[archived downloads](http://downloads.openmicroscopy.org/omero/5.0.4/)
+and
+[archived downloads](http://downloads.openmicroscopy.org/bio-formats/5.0.4/)
+
+For information on upgrading your server, see the upgrade guide for system administrators - http://www.openmicroscopy.org/site/support/omero5/sysadmins/server-upgrade.html
```
Comment (sbesson) [inline _posts/2014-09-02-omero-bio-formats-5-0-4.md]: same as above, needs formatting

File: _posts/2014-09-25-omero-bio-formats-5-0-5.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+title: Release of OMERO & Bio-Formats 5.0.5
+intro-blurb: The OME Consortium is pleased to announce the release of OMERO & Bio-Formats 5.0.5
+---
+Two security vulnerabilities have been discovered in versions of OMERO up to and including 4.4.11
+and up to and including 5.0.4.
+
+**System administrators should review the "Security Vulnerabilities"
+pages on the OME website:**
+
+http://www.openmicroscopy.org/site/products/omero/secvuln
```
Comment (sbesson) [inline _posts/2014-09-25-omero-bio-formats-5-0-5.md]: Hyperlink formatting

File: _posts/2014-09-25-omero-bio-formats-5-0-5.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+An upgrade of all installations is highly recommended. The new versions are available from the
+respective downloads page:
+
+ - [archived downloads](http://downloads.openmicroscopy.org/omero/5.0.5/)
+ - [archived downloads](http://downloads.openmicroscopy.org/omero/4.4.12/)
+
+For information on the upgrade from 4.4.x or 5.0.x to 5.0.5, see the OMERO5 instructions:
+
+https://www.openmicroscopy.org/site/support/omero5/sysadmins/server-upgrade.html
```
Comment (sbesson) [inline _posts/2014-09-25-omero-bio-formats-5-0-5.md]: Hyperlink formatting

File: _posts/2014-09-25-omero-bio-formats-5-0-5.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+For information on the upgrade from 4.4.x to 4.4.12, see the OMERO4 instructions, but upgrading
+directly to 5.0.5 would be preferred:
+
+https://www.openmicroscopy.org/site/support/omero4/sysadmins/server-upgrade.html
```
Comment (sbesson) [inline _posts/2014-09-25-omero-bio-formats-5-0-5.md]: Hyperlink formatting

File: _posts/2014-11-11-omero-bio-formats-5-0-6.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+Two security vulnerabilities have been discovered in versions of OMERO up to and including 5.0.5.
+**System administrators should review the "Security Vulnerabilities" pages on the OME website:**
+
+http://www.openmicroscopy.org/site/products/omero/secvuln
+
+We do not consider either 2014-SV3 CSRF or 2014-SV4 POODLE to be critical vulnerabilities due to the difficulty of exploiting them. **However, we do highly recommend all installations be upgraded.**
+
+The new version is available from the 5.0.6 downloads page - [archived downloads](http://downloads.openmicroscopy.org/omero/5.0.6/)
+
+For information on the upgrade from 4.4.x or 5.0.x to 5.0.6, see the OMERO5 instructions - http://www.openmicroscopy.org/site/support/omero5/sysadmins/server-upgrade.html
```
Comment (sbesson) [inline _posts/2014-11-11-omero-bio-formats-5-0-6.md]: Hyperlink formatting

**Comment by pwalczysko** [comment]
This should fix Seb's comments.

**Comment by sbesson** [comment]
Thanks. Merging.

**Pull request merged**